src/parachain/oracle.ts

```diff
@@ -89,6 +78,11 @@ export interface OracleAPI {
      * @returns Convert a Satoshi amount to Planck
      */
     convertSatoshiToPlanck(satoshi: PolkaBTC): Promise<Big>;
+    /**
+     * @returns The period of time (in milliseconds) after an oracle's last submission
+     * during which it is considered online
+     */
+    getMaxDelay(): Promise<number>;
```

Comment:
I would name this something like `getOnlineTimeout`, what do you think?

Reply:
it's in line with the name on the chain, so I think fine to keep

Reply:
I agree with @theodorebugnet that it's more suggestive. We could change the naming in the parachain

Reply:
ok, agree!